README.md

```diff
@@ -3,8 +3,43 @@
 This is a repository of single-software images to build and push in the `cpg-common` artifact registry.
```

Comment:
I think it's worth outlining here what this repository is _not_ good for: any sort of custom image that's not a single tool, or combining multiple tools into one image that's not generally reusable across projects.